applications/nrf5340_audio/src/modules/lc3_streamer.c

```diff
@@ -350,6 +350,11 @@ int lc3_streamer_stream_register(const char *const filename, uint8_t *const stre
 	bool free_slot_found = false;
 
 	for (int i = 0; i < ARRAY_SIZE(streams); i++) {
+		if (i > UINT8_MAX) {
+			LOG_ERR("Stream index overflow");
+			return -EOVERFLOW;
+		}
+
```

Comment:
This seems unnecessary to check per iteration of the for loop. it should be enough to check `i <= UINT8_MAX` just above `*streamer_idx = i`. Alternatively we could have a build assert that checks if `CONFIG_SD_CARD_LC3_STREAMER_MAX_NUM_STREAMS < UINT8_MAX`

Reply:
See change. I think the build assert is the best option